src/Core/Types.Sorting/SortMiddleware.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using HotChocolate.Language;
+using HotChocolate.Resolvers;
+using HotChocolate.Types.Relay;
+
+namespace HotChocolate.Types.Sorting
+{
+    public class SortMiddleware<T>
+    {
+        private readonly FieldDelegate _next;
+
+        public SortMiddleware(
+            FieldDelegate next)
+        {
+            _next = next ?? throw new ArgumentNullException(nameof(next));
+        }
+
```

Comment:
@gmiserez  can you delete this empty line so that we always only have one empty line between things.